docs/devel/development.md

```diff
@@ -230,9 +230,9 @@ separate dependency updates from other changes._
 
 ```sh
 export KPATH=$HOME/code/kubernetes
-mkdir -p $KPATH/src/k8s.io/kubernetes
-cd $KPATH/src/k8s.io/kubernetes
-git clone https://path/to/your/fork .
+mkdir -p $KPATH/src/k8s.io
+cd $KPATH/src/k8s.io
+git clone https://path/to/your/fork
```

Comment:
I also changed https://path/to/your/fork to https://path/to/your/kubernetes/fork to make it explicit.